### PR TITLE
Add OpenEnergyMonitor Thermostat docs.

### PR DESCRIPTION
**Pull request by Cadair**
**Description:**

Documentation for the OpenEnergyMonitor thermostat.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#5583

### CODE REVIEW

File: source/_components/climate.openenergymonitor.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+therefore there are configuration options for the initial setpoint and the away
+setpoint.
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate oem:
+  - platform: oem
+    name: Home
```
Comment (fabaff) [inline source/_components/climate.openenergymonitor.markdown]: Is missing the var description.

File: source/_components/climate.openenergymonitor.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    host: 192.168.0.100
+    port: 80
+    target_temp: 19
+    away_temp: 14
+```
```
Comment (fabaff) [inline source/_components/climate.openenergymonitor.markdown]: Please keep this sample minimized. This avoid issues caused by copy-&-paste for users. If needed, add the full example at the end of the page.

File: source/_components/climate.openenergymonitor.markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+date: 2017-01-27 11:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Climate
+---
+
+This platform supports the ESP82666 based "WiFi MQTT Relay / Thermostat" sold
+by
```
Comment (fabaff) [inline source/_components/climate.openenergymonitor.markdown]: No need for line breaks. The renderer will take care about this.

File: source/_components/climate.openenergymonitor.markdown [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Open Energymonitor WiFi Thermostat"
+description: "Instructions how to integrate the Open Energy Monitor thermostat with Home Assistant."
+date: 2017-01-27 11:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Climate
```
Comment (fabaff) [inline source/_components/climate.openenergymonitor.markdown]: Please add `ha_release: 0.38`.

**Comment by armills** [comment]
Also, don't forget to rename this to climate.oem.markdown.

**Pull request merged**